src/public/styles.css

```diff
@@ -13,3 +13,12 @@
 #limit-select {
     max-width: 280px;
 }
+
+@media screen and (min-width: 680px) {
+    body {
+        background-image: url('RedisLabs_Illustration.svg');
```

Comment:
I think all urls should be started with './' or even '/'

Reply:
done